ynr/apps/parties/importer.py

```diff
@@ -422,6 +423,26 @@ def download_emblem(self):
             f.write(r.content)
         return ntf.name
 
+    def clean_image(self, image_file_name):
+        # Normalize the image: Sometimes the EC publishes png images with an
+        # alpha channel. This breaks later conversion to JPEG.
+        # See: https://github.com/jazzband/sorl-thumbnail/issues/564
+        with Image.open(image_file_name) as img:
+            # Convert RGBA to RGB
+            if img.mode == "RGBA":
+                # Create a white background and paste the image onto it
+                background = Image.new("RGB", img.size, (255, 255, 255))
+                background.paste(img, mask=img.split()[3])
+                img = background
+            # # LA is luminance + alpha. e.g. greyscale with transparency
```

Comment:
most minor of minor nitpicks

```suggestion
            # LA is luminance + alpha. e.g. greyscale with transparency
```